src/ocm/components/clusterDetail/ProgressBarAlerts.tsx

```diff
@@ -69,8 +69,19 @@ export const HostInstallationWarning = ({
       >
         <RenderIf condition={failedHosts !== 0}>
           <p>
-            {failedHosts}/{totalHosts} {pluralize(totalHosts, hostsType)} {message} Due to this, the
-            cluster will be degraded. Please check the installation log for more information.
+            {hostsType === 'worker' ? (
+              <>
+                Error information for each host can be found in the host inventory table. To retry
+                adding failed hosts, navigate to the clusters list inside OpenShift Cluster Manager,
+                click your cluster name, and select the <b>Add hosts</b> tab.
+              </>
+            ) : (
+              <>
+                {failedHosts}/{totalHosts} {pluralize(totalHosts, hostsType)} {message} Due to this,
+                the cluster will be degraded. Please check the installation log for more
+                information.
+              </>
+            )}
```

Comment:
What if the user is using the standalone/podman-deployed/offline UI?
No "add hosts tab" in that case...

Reply:
What should be displayed in that case? The second message as well?

Reply:
Standalone has the option to transform to day2 and add hosts, right? Could we have something like this?
![image](https://user-images.githubusercontent.com/87187179/190180476-ba6de4b5-6c9d-419f-a68d-2d1acb268591.png)

Reply:
Sounds good to me.